package/Dockerfile.dapper-base

```diff
@@ -40,4 +42,6 @@ RUN dnf -y distrosync && \
     tar -xzf kubefedctl-${KUBEFED_VERSION}-linux-${ARCH}.tgz && cp kubefedctl /usr/bin/ && chmod a+x /usr/bin/kubefedctl && \
     curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
     GOFLAGS="" go get -v github.com/onsi/ginkgo/ginkgo && \
-    GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports
+    GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports && \
```

Comment:
May be we can also remove the downloaded sources for that, what do you think?, we only want the binary, as other deps come via "vendor" directory later.

Reply:
I’d rather upgrade to Fedora 31 and replace this with the `golang-x-tools-goimports` package ;-).